BackgroundChanger.js

```diff
@@ -0,0 +1,31 @@
+const TIME = 10;
+
+function generateColors() {
+  const colors = {};
+  const randomColor = () => {
+    const r = Math.floor(Math.random() * 256);
+    const g = Math.floor(Math.random() * 256);
+    const b = Math.floor(Math.random() * 256);
+    return `rgb(${r},${g},${b})`;
+  };
+
+  for (let i = 0; i < TIME; i++) {
+    let color = randomColor();
+    while (color in colors) {
+      color = randomColor();
+    }
+    colors[color] = true;
```

Comment:
Unless I'm not understanding your comments correctly, then no, the colors will never collide here.

When choosing a random set of colors it uses a `while` loop to check to make sure that color is not already within the `map`, and if it is, then it will keep generating a random color. Once it is unique, breaks out of the `while` loop and the color is added to the `map`.